### PR TITLE
Apply async-await form

### PR DESCRIPTION
**Pull request by d3m3vilurr**
these codeset isn't tested yet.

### CODE REVIEW

File: src/base.js [hunk 1]
```diff
@@ -510,107 +506,87 @@ class Base {
...
-        warn('ignoring failed attempt to give puppet client admin on the status room');
-      }).then(()=> {
-        return room_id;
-      });
-    };
 
-    return puppetClient.getRoomIdForAlias(roomAlias).then(({room_id}) => {
+    let matrixRoomId;
+    try {
+      const { room_id } = puppetClient.getRoomIdForAlias(roomAlias);
```
Comment (AndrewJDR) [inline src/base.js]: Add await here :)

**Comment by AndrewJDR** [comment]
I am bravely testing this on one of my bridges :)
So far, just noticed a missing await (which did cause an issue -- I added a note in the code review above). Will keep an eye out for other problems.

**Comment by AndrewJDR** [comment]
Another person should test this, maybe @thomas-profitt ?

**Comment by d3m3vilurr** [comment]
Thank you fix my mistake :+1:

**Comment by thomas-profitt** [comment]
I'm testing this in "production" now. I'll give it a few days.

**Comment by d3m3vilurr** [comment]
least 3 people tested. imo nobody test to generate `registration.yml` but it would be work.

**Pull request merged**